fpm/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+bundle install --path "${HOME}/bundles/${JOB_NAME}"
+
+cd "recipes/${RECIPE_NAME}/"
+
+bundle exec fpm-cook --target deb
+
+rsync -v pkg/*.deb deploy@apt-1.management:/tmp
+
```

Comment:
I'm not totally sure what that would look like. The Jenkins workspace is cleared on each run so I believe there's no way to figure out what's already been built, and the `/tmp` directory might not persist across reboots.